templates/entity-add-list.html.twig

```diff
@@ -0,0 +1,32 @@
+{#
+/**
+ * @file
+ * Default theme implementation to present a list of available bundles.
+ *
+ * Available variables:
+ *   - class: The entity type specific class. E.g. 'node-add-list'.
+ *   - create_bundle_url: The url to the bundle creation page.
+ *   - bundle_type_label: The lowercase label of the bundle entity type.
+ *   - bundles: A list of bundles, each with the following properties:
+ *     - add_link: link to create an entity of this bundle.
+ *     - description: Bundle description.
+ *
+ * @see template_preprocess_entity_add_list()
+ *
+ * @ingroup themeable
+ */
+#}
+{% if bundles is not empty %}
+  <dl class="entity-add-list {{ class }}">
```

Comment:
yeah we should have an attributes object + define the classes at the top of the file